src/components/ex-live-search-results/ex-live-search-results.scss

```diff
@@ -0,0 +1,101 @@
+.ex-live-search-results {
+  @apply overscroll-contain absolute hidden overflow-y-auto bg-white;
+
+  box-shadow: 0px 3px 45px #0000002c;
+  max-height: calc(100vh - 200px);
+  top: 100%;
+  width: 120%;
```

Comment:
I would even go as far as to increase the width to an equivalent of 600px to give users a nice overview:

![image](https://user-images.githubusercontent.com/88790097/141470363-1aa72740-e184-4b13-8cf8-67544c2b504d.png)

Reply:
changed to 600px